modules/metering-exposing-the-reporting-api.adoc

```diff
@@ -36,10 +37,11 @@ curl -H "Authorization: Bearer $TOKEN" -k "https://$METERING_ROUTE_HOSTNAME/api/
 
 Be sure to replace the `name=[Report Name]` and `format=[Format]` parameters in the URL above. The `format` parameter can be json, csv, or tabular.
 
+[id="metering-authenticate-using-username-password_{context}"]
 === Authenticate using a username and password
 We are able to do basic authentication using a username and password combination, which is specified in the contents of a htpasswd file. By default, we create a secret containing an empty htpasswd data. You can, however, configure the `reporting-operator.spec.authProxy.htpasswd.data` and `reporting-operator.spec.authProxy.htpasswd.createSecret` keys to use this method.
 
-Once you have specified the above in your `MeteringConfig`, you can run the following command:
+Once you have specified the above in your MeteringConfig, you can run the following command:
```

Comment:
s/Once you have specified the above in your MeteringConfig/After you specify the required values in the MeteringConfig definition